canmatrix/dbc.py

```diff
@@ -409,6 +410,8 @@ def load(f, **options):
     else:
         dbcCommentEncoding = dbcImportEncoding
 
+    float_factory = options.get('float_factory', decimal.Decimal)
```

Comment:
Perhaps we should default to `decimal.Decimal` but doing it here would override the default of `float` in the `Signal` constructor.  If we like this `float_factory` interface and are going to spread it around, it should probably be consistent.

https://github.com/ebroecker/canmatrix/blob/a838e833befd666b32703bcf41fd2f0fa575c7a1/canmatrix/canmatrix.py#L201